app/controllers/users_controller.rb

```diff
@@ -86,6 +88,17 @@ def update
     end
   end
 
+  # PUT /users/device-token
+  def save_device_token
+    args = params.require :token
+    DeviceToken.new(token: args).save
+
+    respond_to do |f|
+      f.html
+      f.json { render json: nil, status: :ok }
+    end
```

Comment:
I don't think you need the whole respond_to block, just what is inside the brackets at line 98.